changelog/v0.12.0/gomod.yaml

```diff
@@ -0,0 +1,4 @@
+changelog:
+- type: BREAKING_CHANGE
```

Comment:
why is this a breaking change?

Reply:
We decided it was a breaking change because it completely changes the way it functions

Reply:
the description for the PR is confusing/incomplete. can you give a more detailed account?